package.json

```diff
@@ -15,7 +15,7 @@
     "extract-text-webpack-plugin": "^0.3.8",
     "file-loader": "^0.8.1",
     "flux": "^2.0.1",
-    "html-webpack-plugin": "ampedandwired/html-webpack-plugin#1.4.0",
+    "html-webpack-plugin": "~1.4.0",
```

Comment:
Here "^1.4.0" or ">=1.4.0" would be better since the package must be at least version 1.4.0.